### PR TITLE
Export getArgType

### PR DESCRIPTION
**Pull request by Leospard**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by wongoo** [comment]
@Leospard why it's need to export getArgType? What will u use it for?

File: request.go [hunk 1]
```diff
@@ -33,6 +33,10 @@ import (
 // dubbo
 /////////////////////////////////////////
 
+func GetArgType(v interface{}) string {
```
[the remark this answers is not in the document]
Comment (Leospard) [reply request.go:36]: 当时直观想着就是包装一下 Export 出来给 triple encoding 使用就行，不用修改其他的了。也可以直接改下面的，直接修改更好是嘛，我自己不是很确定

**Comment by Leospard** [comment]
> @Leospard why it's need to export getArgType? What will u use it for?

It will be used in triple. Previously encoding/java_type.go just copy all the code of this method, and I wanna to wrap an exported method to avoid maybe some unexpected problems in the future.

File: request.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (Leospard) [reply request.go:36]: 👌 fix

**Comment by tiltwind** [comment]
@Leospard u should not use dubbo protocol in this library, use [that](https://github.com/apache/dubbo-go/blob/d70b6c0db44b4790ff4de1be8936162817be8080/protocol/dubbo/hessian2/hessian_request.go#L35) in dubbo-go instead. The dubbo protocol code has already bean moved to dubbo-go project. I think the relative code should be marked as deprecated. This project will focus on the implement of hessian2 serilization

**Comment by tiltwind** [comment]
see https://github.com/apache/dubbo-go-hessian2/pull/339/files

**Pull request closed without merging**